vizro-core/docs/pages/user_guides/custom_charts.md

```diff
@@ -3,8 +3,14 @@
 This guide shows you how to create custom charts and how to add them to your dashboard.
 
 The [`Graph`][vizro.models.Graph] model accepts the `figure` argument, where you can enter _any_ [`plotly.express`](https://plotly.com/python/plotly-express/) chart as explained in the user guide on [components][graph].
+In general, the usage of our custom chart decorator `@capture("graph")` is required if your plotly express chart requires any post-update calls or customization.
 
-We always recommend starting with [`plotly.express`](https://plotly.com/python/plotly-express/) charts first, but in case that none of the available charts fulfill your requirements, you can also use any custom created [`plotly.graph_objects.Figure()`](https://plotly.com/python/graph-objects/) object (in short `go.Figure()`). It is equally possible to _enhance_ the resulting `go.Figure()` of a `plotly.express` function call. In general, custom/customized charts need to obey the following conditions:
+**When to use our custom chart decorator?**
```

Comment:
Hmm.. I kind of disagree. I don't know how it's commonly done in the docs but for me, admonitions are helpful if you want to provide additional info outside the main content. Admonitions will then help to embed that side information without disrupting the flow.

So it depends on how you consider the content you embed in there, and for me, this is not side information but main content.

Reply:
Hm I think it depends on the admonitions, they can also be use to high-light things, as e.g. tried on the below bit, which is the key information.

In this particular case I think the bold looks a little out of place, as even the page header is not bold, maybe a section header would be better? 🤔  And then it also repeats the information of the paragraph before just in a more detailed way. So I thought an expandable admonition might be good, as it allows you to deep dive on the content mentioned before

Reply:
I think we tend to overuse admonitions a bit 😄 The section below for me could also just be main content. But I've removed the bold text now. I still prefer to have it as main text though, as otherwise we would have an admonition after an admonition.

Reply:
Yes agree, let's maybe get rid of the admonition below as well!